index.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const differenceWith = require('lodash/differenceWith');
+const isMatch = require('lodash/isMatch');
+const isPlainObject = require('lodash/isPlainObject');
+
+const waitForActions = (store, actions, timeout) => {
+    timeout = timeout || 50;
+
+    if (typeof actions === 'string' || isPlainObject(actions)) {
+        actions = [actions];
+    }
+
```

Comment:
Could you remove this line? I'm being a bit picky but they kind of belong to the same "normalization"